src/app/app-routing.module.ts

```diff
@@ -5,12 +5,15 @@ import { GameCenterComponent } from './components/game-center/game-center.compon
 import { RecordComponent } from './components/record/record.component';
 
 const routes: Routes = [
-  {path: 'game', component: GameCenterComponent},
-  {path: 'userRecord', component: RecordComponent},
   { path: '', redirectTo: 'login', pathMatch: 'full' },
   { path: 'login', component: LoginComponent },
   { path: 'game-center', component: GameCenterComponent },
+<<<<<<< HEAD
+  {path: 'user-record', component: RecordComponent},
+];
+=======
```

Comment:
```suggestion
  {path: 'user-record', component: RecordComponent},
];
```
U left the characters from the conflict solving